sdks/java/io/xml/build.gradle

```diff
@@ -21,6 +21,15 @@ applyJavaNature()
 
 description = "Apache Beam :: SDKs :: Java :: IO :: XML"
 
+/*
+ * We need to rely on manually specifying these evaluationDependsOn to ensure that
+ * the following projects are evaluated before we evaluate this project. This is because
+ * we are attempting to reference the "sourceSets.test.output" directly.
```

Comment:
<!--thread_id:cc_171364576_t; commit:e3960b9fea070f4d514b71629fad156fc4f8773c; resolved:0-->
<!--section:context-quote-->
> **lgajowy** wrote:
> You are right! We don't need it here - my bad.

<!--section:body-->
Seems like code wasn't updated. Forgot to push ?